saleor/graphql/order/types.py

```diff
@@ -521,7 +521,7 @@ def resolve_discount(root: models.OrderEvent, info):
 
     @staticmethod
     def resolve_status(root: models.OrderEvent, _info):
-        return root.parameters.get("status")
+        return root.parameters.get("status") or ""
```

Comment:
I believe this is not working correctly, cause empty string is not part of `TransactionEventStatus`.
Also I think adding one test for that would be good.

Reply:
In other resolvers, we return None. Why do we return `""` here?

Reply:
@kadewu  good catch. Thanks. Will extend the tests also 👍